src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -212,6 +229,7 @@ public static LanguageVersion MapSpecifiedToEffectiveVersion(this LanguageVersio
             switch (version)
             {
                 case LanguageVersion.Latest:
+                    return LanguageVersion.CSharp7_1;
```

Comment:
I wonder if we could make this code invariant so it doesn't have to be updated with every point release. Maybe we just return the largest enum entry except for `LanguageVersion.Latest`.